### ISSUE TITLE
snap in the ubuntu store is outdated and unsandboxed?

### ISSUE THREAD

**Issue by diracdeltas** [snapcrafters/brave#24]
We received a bug report that a user downloaded Brave on Ubuntu via Snap and got an error that it was running without the sandbox. It appears @posix4e added the sandbox a month ago in https://github.com/snapcrafters/brave/commit/709b55d5aa2b01489b845653480b6cfa93ab767b but https://snapcraft.io/brave hasn't been updated since November. Could you please update the version in the Snap store to avoid exposing users to security issues?

Sorry if this is not the right place to report this

**Comment by diracdeltas** [comment]
Also we would appreciate it if any Snap listings for Brave mentioned that this package is now community maintained and may not be as up-to-date as the apt/rpm repos (which we recommend)

**Comment by posix4e** [comment]
Yep. I think this is my fault. I still don't have the ability to push new snaps to the snapcraft store.

**Comment by diracdeltas** [comment]
it turns out i have access to the Brave snap listing at https://snapcraft.io/brave so i logged in and made some messaging changes to clarify that it's not officially maintained

@posix4e why aren't you able to do updates?

EDIT figured it out in DM

**Comment by posix4e** [comment]
repo forked. forum notified. i will new publishing across all three snap channels for stable beta and edge based on brave’s 3 channels simultaneously as frankly we need everything updated
